.circleci/config.yml

```diff
@@ -27,6 +27,9 @@ executors:
     machine:
       image: ubuntu-1604:201903-01
 
+  cloudify-machine-510:
+    machine:
+      image: ubuntu-1604:201903-01
```

Comment:
Why do we need this?

Reply:
for parallel running. each vm creates its own cfy_manager.

Reply:
ok makes sense